website/ladder.php

```diff
@@ -97,39 +98,39 @@
-			<li><a data-target="push" class="button" href="/ladder/gen8ubers">Ubers</a></li>
-			<li><a data-target="push" class="button" href="/ladder/gen8uu">UnderUsed</a></li>
-			<li><a data-target="push" class="button" href="/ladder/gen8ru">RarelyUsed</a></li>
-			<li><a data-target="push" class="button" href="/ladder/gen8nu">NeverUsed</a></li>
-			<li><a data-target="push" class="button" href="/ladder/gen8pu">PU</a></li>
-			<li><a data-target="push" class="button" href="/ladder/gen8lc">Little Cup</a></li>
-			<li><a data-target="push" class="button nav-last" href="/ladder/gen8monotype">Monotype</a></li>
+			<li><a data-target="push" class="button nav-first" href="/ladder/gen9ou">OverUsed</a></li>
+			<li><a data-target="push" class="button" href="/ladder/gen9ubers">Ubers</a></li>
+			<li><a data-target="push" class="button" href="/ladder/gen9uu">UnderUsed</a></li>
+			<li><a data-target="push" class="button" href="/ladder/gen9ru">RarelyUsed</a></li>
+			<li><a data-target="push" class="button" href="/ladder/gen9nu">NeverUsed</a></li>
+			<li><a data-target="push" class="button" href="/ladder/gen9pu">PU</a></li>
+			<li><a data-target="push" class="button" href="/ladder/gen9lc">Little Cup</a></li>
+			<li><a data-target="push" class="button nav-last" href="/ladder/gen9monotype">Monotype</a></li>
 		</ul>
 		<ul class="laddernav">
-			<li><a data-target="push" class="button nav-first" href="/ladder/gen81v1">1v1</a></li>
-			<li><a data-target="push" class="button" href="/ladder/gen8anythinggoes">Anything Goes</a></li-->
-			<li><a data-target="push" class="button" href="/ladder/gen8zu">ZU</a></li-->
-			<li><a data-target="push" class="button" href="/ladder/gen8battlestadiumsingles">Battle Stadium Singles</a></li>
-			<li><a data-target="push" class="button nav-last" href="/ladder/gen8cap">CAP</a></li>
+			<li><a data-target="push" class="button nav-first" href="/ladder/gen91v1">1v1</a></li>
+			<li><a data-target="push" class="button" href="/ladder/gen9anythinggoes">Anything Goes</a></li-->
+			<li><a data-target="push" class="button" href="/ladder/gen9zu">ZU</a></li-->
+			<li><a data-target="push" class="button" href="/ladder/gen9battlestadiumsingles">Battle Stadium Singles</a></li>
+			<li><a data-target="push" class="button nav-last" href="/ladder/gen9cap">CAP</a></li>
 		</ul>
 		<ul class="laddernav">
-			<li><a data-target="push" class="button nav-first" href="/ladder/gen8randomdoublesbattle">Random Doubles Battle</a></li>
-			<li><a data-target="push" class="button" href="/ladder/gen8doublesou">Doubles OU</a></li>
-			<li><a data-target="push" class="button nav-last" href="/ladder/gen8vgc2021">VGC 2021</a></li>
+			<li><a data-target="push" class="button nav-first" href="/ladder/gen9randomdoublesbattle">Random Doubles Battle</a></li>
+			<li><a data-target="push" class="button" href="/ladder/gen9doublesou">Doubles OU</a></li>
+			<li><a data-target="push" class="button nav-last" href="/ladder/gen9vgc2021">VGC 2021</a></li>
```

Comment:
```suggestion
			<li><a data-target="push" class="button nav-last" href="/ladder/gen9vgc2023regulationc">VGC 2023 Regulation C</a></li>
```

Reply:
in the future, something autocompleting would be preferred but this is fine for now